draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -461,84 +446,55 @@ Coherent pluggable have revolutionized optical networking by offering a powerful
 
```

Comment:
Supported modes are per media interface and not specific to pluggable optical modules. The following description is intended to provide the context. (see https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang and https://datatracker.ietf.org/doc/draft-ietf-ccamp-dwdm-if-param-yang/)